### ISSUE TITLE
Incomplete line numbering of sample exam

### ISSUE THREAD

**Issue by mhamburg** [istqborg/istqb_product_base#157]
Line numbers are missing in the AL-TA sample exam on pages 27, 28, 29, 33

<img width="435" alt="Screenshot 2024-12-30 105945" src="https://github.com/user-attachments/assets/95e1c33b-d58a-48b1-9d31-9b656170724f" />

**Comment by danopolan** [comment]
Is this still valid? I do not see line numbers being generated in [release-version](https://github.com/istqborg/istqb-ctal-ta/tree/release-version) branch despite `line-numbers: true` in all documents.

**Comment by mhamburg** [comment]
Yes it is still valid. I turned line numbers on but did not generate (pushed changes always with [skip ci].

**Comment by mhamburg** [comment]
Here is yet another example generated this morning:

<img width="263" alt="Screenshot 2025-01-12 183405" src="https://github.com/user-attachments/assets/056d3a36-f023-4f40-8c83-bfb83275e17b" />

Note that in this example we also have an orphane heading that was commented by a reviewer.

Now I turned line numbering off because we don't need it in the release version. But the issue is relevant for future Alpha and Beta versions that are sent to the reviewers.

**Comment by danopolan** [comment]
Raised in #167

**Comment by Witiko** [comment]
> @danopolan This is caused by the code from PR https://github.com/istqborg/istqb_product_base/pull/38 that prevents splitting short questions over more pages.
> 
> If you'd like, we can disable the code when line numbers are enabled. The downside is that pages will break differently for reviewers, which increases the risk that an issue with page breaking would go unnoticed during review. I opened PR https://github.com/istqborg/istqb_product_base/pull/169, which implements this change.
> 
> Alternatively, we might also disable to code altogether. This will ensure that pages are broken the same for reviewers. However, short questions may be split over more pages. 

 _Originally posted by @Witiko in [#167](https://github.com/istqborg/istqb_product_base/issues/167#issuecomment-2598478410)_ on January 17

Please, let me know which trade-off seems more reasonable: a) having line numbers in the review copy but potentially different page breaks from the final version or b) having line numbers in the review copy and having the same page breaks as in the final version but short questions being potentially split over two pages.
